spec/features/user_can_pick_joke_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+feature "when I visit home page" do
+  fixtures :jokes
+  before(:each) do
+    visit '/'
+  end
+  it "has a random joke" do
+    expect(page).to have_button('Pick a Joke for me!')
+  end
+
+  it  "redirects to /items" do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.